rest/v9/guild.ts

```diff
@@ -303,6 +304,11 @@ export type RESTPatchAPIGuildChannelPositionsJSONBody = Array<{
  */
 export type RESTPatchAPIGuildChannelPositionsResult = never;
 
+/**
+ * https://discord.com/developers/docs/resources/guild#list-active-threads
+ */
+export type RESTGetAPIGuildThreadsResult = Omit<APIThreadList, 'has_more'>;
```

Comment:
Just a question since phone review, is this route in the routes object?

Reply:
oops, was gonna do it but forgot, will add